types/recharts/index.d.ts

```diff
@@ -821,22 +821,35 @@ export interface TreemapProps extends EventAttributes, Animatable {
 
 export class Treemap extends React.Component<TreemapProps> { }
 
-export interface Label {
-    className?: string;
+export interface parseViewBoxArgs {
```

Comment:
Am I correct that this interface is no longer needed?

Reply:
@mthmulders good catch. Thanks.